### PR TITLE
Switch Repository for Continuing Development of FlowTorch

### PR DESCRIPTION
**Pull request by stefanwebb**
### Motivation
Development of FlowTorch is being continued by the main author outside of its existing home.

### Changes proposed
Remove the GitHub actions to upload the PyPI package and update the website.

Notification in `readme.md` of where to find future development.

### CODE REVIEW

**Comment by jpchen** [comment]
@stefanwebb let me know when this is ready for merge.

**Comment by stefanwebb** [comment]
> @stefanwebb let me know when this is ready for merge.

Thanks @jpchen! It's ready to be looked at :)

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ coverage.xml
 .mypy_cache/
 .vscode/
 /.coverage
+.DS_Store
```
Comment (jpchen) [inline .gitignore:15]: Add a newline.

Comment (stefanwebb) [reply .gitignore]: Done!

**Comment by stefanwebb** [comment]
@jpchen should be good to go now! :)

**Pull request closed without merging**